addons/mail/static/src/components/channel_member/channel_member.xml

```diff
@@ -13,11 +14,12 @@
                             'o-isDeviceSmall': messaging.device.isSmall,
                             'small': !messaging.device.isSmall,
                         }"
+                        hasOpenChat="true"
```

Comment:
only if hasOpenChat

Reply:
I think for all the members except Guest/Archived we will be able to open chat. And Archived users are by default not visible in `members list`. And for Guest users we don't have personaImStatus. So, giving true, will not cause any problems.

Reply:
Guests should have personaImStatus, if not yet it will come soon.